src/elements/content-preview/preview-header/PreviewHeader.js

```diff
@@ -62,7 +62,8 @@ const PreviewHeader = ({
     token,
 }: Props) => {
     const fileId = file && file.id;
-    const shouldRenderAnswers = fileId && contentAnswersProps.show;
+    const { show: showContentAnswers, ...restContentAnswersProps } = contentAnswersProps;
+    const shouldRenderAnswers = fileId && showContentAnswers;
```

Comment:
i don't see harm in passing `show` along with the rest of the contentAnswersProps to ContentAnswers. it could simplify the exporting types and whatnot